src/Configuration/CliOptions.cs

```diff
@@ -71,7 +71,12 @@ public static (PlcSimulation PlcSimulationInstance, List<string> ExtraArgs) Init
                         throw new OptionException("The max OPC string length must be larger than 0.", "opcmaxstringlen");
                     }
                 }
-            },
+            },
+
+            // OTLP configuration
+            { "otlpe|otlpendpoint=", $"the endpoint Uri to which the otlp exporter is going to send traces .\nDefault: {config.OtlpEndpointUri}", (s) => config.OtlpEndpointUri = s },
```

Comment:
>otlpendpoint [](http://example.com/codeflow?start=21&length=12)

Do we need to specify the endpoint? shouldn't the port be enough?

We also have an option to specify the port for the pn.json, we need to figure that out.

Reply:
endpoint is needed for the otel collector.